validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -476,8 +477,8 @@ def run(self):
                                 set(peered_endpoints) -
                                 set([self._endpoint]))
 
-                        LOGGER.debug("Number of peers: %s", len(peers))
-                        LOGGER.debug("Peers are: %s", list(peers.values()))
+                        LOGGER.debug("Number of peers: %s", peer_count)
```

Comment:
These should be on 1 line instead of three. We need less lines, more info. (generally speaking)

Reply:
second